src/velocitas/model_generator/tree_generator/file_formats.py

```diff
@@ -86,5 +87,6 @@ def load_tree(self):
         output_json = json.load(open(self.file_path))
         self.__extend_fields(next(iter(output_json.values())))
         print("Generating tree from json...")
-        tree = vspec.render_tree(output_json)
+        vspec.load_units(self.file_path, list())
```

Comment:
We need to find a solution to link the unit file for our "main case" which is using the json-based VSS catalogue:
* Either we provide a default unit file within our toolchain,
* we also assume a unit.yaml lying beside the json file,
* we offer a way to specify the unit file location explicitly, or
* other proposals ...

Reply:
solution decided and implemented